modules/serverless-logic-job-service-postgresql-ephemeral-config.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-job-service.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-job-service-postgresql-ephemeral-config_{context}"]
+= Job service PostgreSQL and Ephemeral configurations
+
+The Job service supports two types of configurations: PostgreSQL and Ephemeral. Depending on the persistence mechanism you choose, you can configure the Job service accordingly.
+
+[id="serverless-logic-job-service-postgresql-config_{context}"]
+== Job service PostgreSQL configurations
+
+PostgreSQL is the recommended database for the Job Service. It also includes an initialization procedure that integrates Flyway for database setup, automatically managing the database schema. This ensures that the service creates or updates tables as needed.
```

Comment:
@domhanak  maybe we can add which image we tested in QE? cc @kaldesai 
This is the recommended PostgreSQL image: https://catalog.redhat.com/software/containers/rhel9/postgresql-13/61a60a80bfd4a5234d596288

Reply:
@ricardozanini yea I agree, I need to rerun the tests with this one you are mentioning though. Will get back to you with result soon

Reply:
Okay, sure! Let me know about the same